src/tools-evaluation.md

```diff
@@ -14,6 +14,22 @@ of using Linux to boot into an operating system.
 
 ## Root filesystem
 
+Linux needs a root filesystem with at least one binary that is called
+[`init`](https://docs.kernel.org/admin-guide/init.html). Since booting a system
+is a cumbersome task, additional tools aid in both development and investigating
+possible issues.
+
+### Core utilities
+
+Unix already came with lots of little utilities for the user of the system,
+which may be anyone from a system developer to an administrator of a shared or
+provided system, or an end user. Further tools have been created over the years,
+and the [GNU core utilities](https://en.wikipedia.org/wiki/GNU_Core_Utilities)
+are essentially a collection of tools resulting from merging other collections.
+Note that there are still many other utilities that are not part of coreutils.
+At the same time, there are multiple other implementations now, which differ in
+terms of arguments and flags and possibly additional utilities they include.
```

Comment:
I might replace this paragraph with:

```suggestion
There are many sources of shell commands:
```

It isn't necessary to explain everything in detail. If the reader is either a boot chain maintainer or a LinuxBoot developer then I would assume this information is known. I think there are places where prose makes sense like making the argument of UEFI vs. non-UEFI that is not documented well elsewhere. But this is a very practical article.